functions.go

```diff
@@ -8,15 +8,22 @@ import (
 )
 
 const (
-	functionsBasePath      = "/v2/functions/namespaces"
-	functionsNamespacePath = functionsBasePath + "/%s"
+	functionsBasePath        = "/v2/functions/namespaces"
+	functionsNamespacePath   = functionsBasePath + "/%s"
+	functionsTriggerBasePath = functionsNamespacePath + "/triggers"
 )
 
 type FunctionsService interface {
 	ListNamespaces(context.Context) ([]FunctionsNamespace, *Response, error)
 	GetNamespace(context.Context, string) (*FunctionsNamespace, *Response, error)
 	CreateNamespace(context.Context, *FunctionsNamespaceCreateRequest) (*FunctionsNamespace, *Response, error)
 	DeleteNamespace(context.Context, string) (*Response, error)
+
+	ListTriggers(context.Context, string) ([]FunctionsTrigger, *Response, error)
+	GetTrigger(context.Context, string, string) (*FunctionsTrigger, *Response, error)
+	CreateTrigger(context.Context, string, *FunctionsTriggerCreateRequest) (*FunctionsTrigger, *Response, error)
+	UpdateTrigger(context.Context, string, string, *FunctionsTriggerUpdateRequest) (*FunctionsTrigger, *Response, error)
+	DeleteTrigger(context.Context, string, string) (*Response, error)
```

Comment:
I thought I saw a PR for bringing back enable/disable trigger support. Do you intend to add those to `godo` at some point?

Reply:
enable/disable is part of the `UpdateTrigger` request, we will use it in `doctl` to build the command `doctl sls triggers enable <trigger-name>`